zubhub_frontend/zubhub/src/store/actions/activityActions.js

```diff
@@ -0,0 +1,8 @@
+export const setActivities = activities => {
+    return dispatch => {
+      dispatch({
+        type: 'SET_ACTIVITIES',
```

Comment:
Please use constants for action types.

Reply:
Hi @kamthamc  
Thanks again for your guidance 🙏, i will create the action types file with all constants 👍